test/integration/goldens/compute/src/com/google/cloud/compute/v1small/stub/HttpJsonRegionOperationsStub.java

```diff
@@ -54,7 +54,7 @@
  * <p>This class is for advanced usage and reflects the underlying API directly.
  */
 @Generated("by gapic-generator-java")
-@BetaApi("A restructuring of stub classes is planned, so this may break in the future")
+@BetaApi
```

Comment:
Is removing of the comment intentional? If the original comment doesn't make sense for HttpJson, do we need some additional comment to explain why this is BetaApi?

Reply:
I could not come up with a good description except saying that it is in beta (but annotaiton itself is already called BetaApi), that is why I did not give any descriptions. The gist of it is "`REST transport functionality is in beta`" If you have a good ideas how to convey that message in the annotation properly, please suggest